lib/ext/promise.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
```

Comment:
This can likely be removed.  Looks like it is only required in two places:

* tests/helpers/acceptance.js
* tests/helpers/run-command.js

In both cases, I think `var Promise = require('rsvp').Promise;` should work.